test/integ/cli/test_cli_incremental_download.py

```diff
@@ -0,0 +1,596 @@
+    ) -> Tuple[bool, str]:
+        """Wait for a job to complete. Returns (isSuccess, final_status)."""
+        start_time = time.time()
+        while time.time() - start_time < timeout:
+            job = self.deadline_client.get_job(
+                farmId=self.farm_id, queueId=self.queue_id, jobId=job_id
+            )
+
+            task_run_status = job.get("taskRunStatus")
+
+            if task_run_status == "SUCCEEDED":
+                return True, task_run_status
+            elif task_run_status in ["FAILED", "CANCELED"]:
+                return False, task_run_status
+
+            time.sleep(poll_interval)
+
+        return False, "TIMEOUT"
+
+    def run_incremental_download_without_storage_profiles(
+        self,
+        output_dir: str,
+        force_bootstrap: bool = False,
+        conflict_resolution: Optional[str] = None,
+        lookback_window: Optional[int] = None,
+        test_name: Optional[str] = None,
+    ) -> subprocess.CompletedProcess:
+        """Run the incremental download CLI command."""
+        # Use test-specific checkpoint directory to avoid conflicts between parallel tests
+        checkpoint_suffix = f"_{test_name}" if test_name else ""
+        checkpoint_dir = os.path.join(output_dir, f"checkpoints{checkpoint_suffix}")
+        os.makedirs(checkpoint_dir, exist_ok=True)
+
+        cmd = [
+            "deadline",
+            "queue",
+            "incremental-output-download",
+            "--farm-id",
+            self.farm_id,
+            "--queue-id",
+            self.queue_id,
+            "--checkpoint-dir",
+            checkpoint_dir,
+            "--ignore-storage-profiles",
+        ]
+
+        if force_bootstrap:
+            cmd.append("--force-bootstrap")
+
+        if conflict_resolution:
+            cmd.extend(["--conflict-resolution", conflict_resolution])
+
+        if lookback_window is not None:
+            cmd.extend(["--bootstrap-lookback-minutes", str(lookback_window)])
+
+        env = os.environ.copy()
+        env["ENABLE_INCREMENTAL_OUTPUT_DOWNLOAD"] = "1"
+
+        # Run from workspace directory so files are downloaded to their manifest paths
+        # The CLI will create the necessary directory structure based on job manifests
+        return subprocess.run(
+            cmd, capture_output=True, text=True, check=False, env=env, cwd=Path.cwd()
+        )
+
+
+@pytest.fixture(scope="session")
+def incremental_download_test(deadline_cli_test: DeadlineCliTest):
+    """Fixture to get the IncrementalDownloadTest object."""
+    return IncrementalDownloadTest(
+        farm_id=deadline_cli_test.farm_id, queue_id=deadline_cli_test.queue_id
+    )
+
+
+@pytest.fixture(scope="session", autouse=True)
+def cleanup_test_files():
+    """Post-session fixture to cleanup any dangling files from incremental download tests."""
+    yield  # Run all tests first
+
+    # Cleanup patterns from test analysis
+    workspace = Path.cwd()
```

Comment:
It would be better for these tests to create a temporary directory instead of using the CWD. By using the CWD, they can write files to locations that the dev running the tests does not expect.